package ru.ozi_blog.task.exceptions;

/**
 * A ProcessingCalculatorException is thrown if a calculation operation is
 * incorrect
 * 
 * @see ru.ozi_blog.task.calc.Calculator
 * 
 * @author Zhukavets Aleh
 * 
 */
public class ProcessingCalculatorException extends CalculatorException {

}
